### ISSUE TITLE
Blank index page

### ISSUE THREAD

**Issue by Sh1gy** [mjnaderi/Sharif-Judge#85]
![16575340520155087883374899519981](https://user-images.githubusercontent.com/95503865/178241227-a789c68a-f102-4993-a22b-d06337720259.jpg) 
This is what I'm getting what am I missing

**Comment by flitzcore** [comment]
I have the same issue. Have you found the issue?

**Comment by Sh1gy** [comment]
yes it was because the system was failing to seed the database so i used an
earlier version of xampp and it went away
